keylime/ima/ima.py

```diff
@@ -584,8 +584,8 @@ def unbundle_ima_policy(allowlist_bundle, verify: bool = True) -> Dict[str, Any]
                 message="Verifier is enforcing signature validation, but no public key was provided.", code=405
             )
 
-        allowlist_sig = base64.b64decode(allowlist_bundle.get("sig"))
-        allowlist_sig_key = base64.b64decode(allowlist_bundle.get("key"))
+        allowlist_sig = base64.b64decode(cast(bytes, allowlist_bundle.get("sig")))
+        allowlist_sig_key = base64.b64decode(cast(bytes, allowlist_bundle.get("key")))
 
         if not signing.verify_signature(allowlist_sig_key, allowlist_sig, allowlist_raw):
```

Comment:
seems that `verify_signature` from `keylime/signing.py` is missing the return type

Reply:
I just added type-annotations for signing.py to the other PR.